src/app/clusters/laundry-dryer-controls-server/laundry-dryer-controls-server.cpp

```diff
@@ -163,6 +163,10 @@ Status MatterLaundryDryerControlsClusterServerPreAttributeChangedCallback(const
     {
     case Attributes::SelectedDrynessLevel::Id: {
         uint8_t drynessLevelIdx = 0;
+        if (NumericAttributeTraits<uint8_t>::IsNullValue(*value))
```

Comment:
Why are you de-referencing `value`, when `value` is `uint8_t * value`? This doesn't seem right to me.

I am looking at `NumericAttributeTraits<uint8_t>::IsNullValue`, a little more, but posting comment in case there is an answer someone knows for sure

Reply:
NVM this is correct kNullValue == 0xFF